pkgs/applications/audio/musly/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, cmake, eigen, libav_all }:
+stdenv.mkDerivation rec {
+  name = "musly-${version}";
+  version = "0.2";
+  src = fetchFromGitHub {
+    owner = "dominikschnitzer";
+    repo = "musly";
```

Comment:
`repo = pname;`

Reply:
I prefer leaving that one unabstracted so that the `fetchFromGitHub` block is copiable on its own.